comms/DiskRepository.cpp

```diff
@@ -138,6 +138,10 @@ class CDiskRepository : public CRepositoryBase
                 pos = moduleName.rfind(_T("\\"));
                 moduleName = moduleName.substr(0, pos);
                 algo::replace_all(moduleName, _T("\\"), _T("."));
+                if (!module.empty() && !boost::starts_with(moduleName, module))
```

Comment:
Is "starts_with" appropriate here?

Reply:
It is correct. See below:

![findstartswith](https://user-images.githubusercontent.com/10133951/38033815-e19dd3d6-326e-11e8-9b34-100154819d64.png)

Reply:
@GordonSmith we can restrict it to the selected folder only and no subfolders of the selected folder. Is that the behavior we want or do we want to include the subfolder files as well?